test/expected/chunk_utils.out

```diff
@@ -136,11 +142,71 @@ WHERE h.schema_name = 'public' AND (h.table_name = 'drop_chunk_test1' OR h.table
  _timescaledb_internal | _hyper_3_18_chunk | table | default_perm_user
 (18 rows)
 
+-- next two calls of show_chunks should give same set of chunks as above when combined
+SELECT show_chunks('drop_chunk_test1');
```

Comment:
add an order by name clause same below. This makes these tests more robust.

Reply:
added sorting internally in C